LINUX_COMPATIBILITY_POLICY.md

```diff
@@ -0,0 +1,33 @@
+# Linux compatibility policy
+The GitButler project aims to provide as good a level of compatibility as possible with the various Linux distributions out there. This document outlines the current level of commitment toward different Linux distributions. GitButler _should_ run well on most modern Linux distributions if built specifically for them, but compatibility is only regularly monitored for the officially supported distributions. 
```

Comment:
"as good a level of compatibility as possible with" - would "best-effort" be clearer? Or is there another way of writing this that sounds less like a truism?